README.md

```diff
@@ -1 +1,164 @@
-# afctl
\ No newline at end of file
+# afctl
+
+The proposed CLI tool is being authored to make creating and deployment of airflow projects faster and smoother. 
+As of now, there is no tool out there that can empower the user to create a boilerplate code structure for airflow 
+projects and make development + deployment of projects seamless.
+
+## Installation
+
+Clone the repository. <br />
+Create a new python virtualenv. You can use the following command. <br />
+```bash
+python3 -m venv <name>
+```
+Activate your virtualenv<br/>
+```bash
+source /path_to_venv/bin/activate
+```
+
+Step into the afctl directory and run <br/>
+```bash
+pip3 install .
+```
+
+## Requirements
+Python 3.5+
+
+## Usage
+
+Commands right now supported are
+* init
+* config
+* deploy
+* list
+* generate
+
+To learn more, run 
+```bash
+afctl <command> -h
+```
+<br>
+
+#### Initialize a new afctl project. 
+The project is created in your present working directory. Along with this a configuration file with the same name is 
+generated in **/home/afctl_configs** directory.
+
+
+```bash
+afctl init “name of the project”
+```
+
+* Creates a new project directory.
+* Creates a config file in the home directory
+
+```bash
+afctl init .
+```
+* Initialize the current directory as a project
+* If the directory is already a git repository then the global configs will get automatically set.
+<br>
+
+#### Manage configurations
+
+The configuration file is used for deployment.
+```yaml
+global:
+  git:
+    origin:
+deployment:
+  qubole:
+  docker-local:
+    compose:
+```
+<br>
+
+```
+TYPES:
+   add - add a config for your deployment.
+   update - update an existing config for your deployment.
+       Arguments:
+           -d : Deployment Type
+           -p : Project
+            [ Qubole ]
+               -n : name of connection
```

Comment:
connection -> deployment

Reply:
Deployment is qubole, local, etc.
Connection are subsets of deployments. You can have 2 connections in qubole. One for api and one for us.